Gemfile.lock

```diff
@@ -62,8 +62,6 @@ GEM
     bootsnap (1.4.5)
       msgpack (~> 1.0)
     builder (3.2.4)
-    bulma-rails (0.8.0)
```

Comment:
Is this gem being used?

Reply:
no, i later changed from this to bulma cdn based theme. viz why i removed it